tests/conftest.py

```diff
@@ -20,10 +20,127 @@
 
 import pytest
 
+from iqm.cirq_iqm import IQMDevice, IQMDeviceMetadata
+from iqm.iqm_client import QuantumArchitectureSpecification
+
 existing_run = UUID('3c3fcda3-e860-46bf-92a4-bcc59fa76ce9')
 missing_run = UUID('059e4186-50a3-4e6c-ba1f-37fe6afbdfc2')
 
 
 @pytest.fixture()
 def base_url():
     return 'https://example.com'
+
+
+@pytest.fixture()
+def fake_spec_with_resonator():
+    ndonis_architecture_specification = {
+        'name': 'Ndonis',
+        'operations': {
+            'cz': [
+                ['QB1', 'COMP_R'],
+                ['QB2', 'COMP_R'],
+                ['QB3', 'COMP_R'],
+                ['QB4', 'COMP_R'],
+                ['QB5', 'COMP_R'],
+                ['QB6', 'COMP_R'],
+            ],
+            'prx': [['QB1'], ['QB2'], ['QB3'], ['QB4'], ['QB5'], ['QB6']],
+            'move': [
+                ['QB1', 'COMP_R'],
+                ['QB2', 'COMP_R'],
+                ['QB3', 'COMP_R'],
+                ['QB4', 'COMP_R'],
+                ['QB5', 'COMP_R'],
```

Comment:
There is no move ['QB6', 'COMP_R']. Is that on purpose?

Reply:
Yes, because there might be a device that does not have all moves calibrated. It is used in test_iqm_device.test_qubit_connectivity.